dotnet-6/Models/User.cs

```diff
@@ -1,18 +1,18 @@
-namespace CrudApp.Models;
-
-public class User {
-    public long Id { get; set; }
-    public string Email { get; set; }
-    public string Name { get; set; }
-
-    public User(long Id, string Email, string Name) {
-        this.Id = Id;
-        this.Email = Email;
-        this.Name = Name;
-    }
-
-    public override string ToString()
-    {
-        return $"User: {Id}, {Email}, {Name}";
-    }
-}
\ No newline at end of file
+namespace CrudApp.Models;
+
+public class User
+{
+    public long Id { get; set; }
+    public string Email { get; set; } = string.Empty;
+    public string Name { get; set; } = string.Empty;

```

Comment:
Why do we need to initialize them to empty when there is no empty constructor?

Reply:
We do not, indeed, I removed that 😄